### PR TITLE
Remove unnecessary aws sdk files from installers

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #57:*

*Description of changes:*

- Turn off custom memory management and run-time type information while building aws sdk
- Building aws sdk separately to remove unwanted files from installers
- Updating driver name in installers as the maximum length for DSN name is 32 chars in windows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/IntegrationTests/ITODBCAwsAuth/CMakeLists.txt [hunk 1]
```diff
@@ -21,11 +21,13 @@ include_directories(	${UT_HELPER}
...
+						)
 
 # Generate executable
 add_executable(itodbc_aws_auth ${SOURCE_FILES})
 
 # Library dependencies
-target_link_libraries(itodbc_aws_auth elasticodbc itodbc_helper ut_helper gtest_main aws-cpp-sdk-core)
+target_link_libraries(itodbc_aws_auth elasticodbc itodbc_helper ut_helper gtest_main)
 target_compile_definitions(itodbc_aws_auth PUBLIC _UNICODE UNICODE)
+target_link_libraries(itodbc_aws_auth aws-cpp-sdk-core)
```
Comment (jordanw-bq) [inline src/IntegrationTests/ITODBCAwsAuth/CMakeLists.txt:33]: Is there a reason this needs to be on a separate line now?

Comment (rupal-bq) [reply src/IntegrationTests/ITODBCAwsAuth/CMakeLists.txt]: No specific reason. Combined in a single line.

File: src/IntegrationTests/ITODBCAwsAuth/CMakeLists.txt [hunk 2]
```diff
@@ -21,11 +21,12 @@ include_directories(	${UT_HELPER}
...
+						${AWSSDK_INCLUDE_DIR}
+						)
 
 # Generate executable
 add_executable(itodbc_aws_auth ${SOURCE_FILES})
 
 # Library dependencies
 target_link_libraries(itodbc_aws_auth elasticodbc itodbc_helper ut_helper gtest_main aws-cpp-sdk-core)
-target_compile_definitions(itodbc_aws_auth PUBLIC _UNICODE UNICODE)
+target_compile_definitions(itodbc_aws_auth PUBLIC _UNICODE UNICODE)
```
Comment (jordanw-bq) [inline src/IntegrationTests/ITODBCAwsAuth/CMakeLists.txt:32]: Nit: Fix whitespace change here?

Comment (rupal-bq) [reply src/IntegrationTests/ITODBCAwsAuth/CMakeLists.txt]: fixed

**Pull request merged**